test/apitests/data_scenarios/source/v1/Untitled-1.js

```diff
@@ -0,0 +1,56 @@
+
+const destinationName = 'MARKETO';
+const definition = await DestinationDefinitionRepository.findOne({
+    name: destinationName,
+});
+
+if (!definition) {
+    throw new NotFoundError(`${destinationName} destination not found`);
+}
+
+const destinations = await DestinationRepository.findMany(
+    { destinationDefinitionId: definition.id, deleted: false },
+    ['updatedBy'],
+);
+
+const outputDestinations: Struct<Destination>[] = [];
+
+for (const destination of destinations) {
+    const { config } = destination;
+    const rudderEventsMapping: { event: any; marketoPrimarykey: any; marketoActivityId: any; }[] = [];
+    const activityPrimaryKeyMap: { [key: string]: string } = {};
+    const activityEventMap: { [key: string]: string } = {};
+    if (config.customActivityPrimaryKeyMap) {
+        config.customActivityPrimaryKeyMap.forEach((item: { from: string; to: string }) => {
+            activityPrimaryKeyMap[item.from] = item.to;
+        });
+    }
+    if (config.customActivityEventMap) {
+        config.customActivityEventMap.forEach((item: { from: string; to: string }) => {
+            activityEventMap[item.from] = item.to;
+        });
+    }
+    Object.keys(activityEventMap).forEach(event => {
```

Comment:
This is config migration script wrongly added

Reply:
Yeah will remove it